libraries/aws_iam_users.rb

```diff
@@ -0,0 +1,43 @@
+# author: Alex Bedley
+# author: Steffanie Freeman
+# author: Simon Varlow
+# author: Chris Redekop
+class AwsIamUsers < Inspec.resource(1)
+  name 'aws_iam_users'
+  desc 'Verifies settings for AWS IAM users'
+  example ''
+
+  filter = FilterTable.create
+  filter.add_accessor(:where)
+        .add_accessor(:entries)
+        .add(:exists?) { |x| !x.entries.empty? }
```

Comment:
This is where we can add methods to the user-collection as a whole.  Exists struck me as immediately useful, but we might need to add more as we go...